packages/fxa-content-server/app/scripts/views/sms_send.js

```diff
@@ -31,11 +33,19 @@ class SmsSendView extends FormView {
   mustAuth = true;
   template = Template;
 
+  events = assign(this.events, {
+    'click .btn-not-now': preventDefaultThen('gotoProductPage'),
+  });
+
   getAccount() {
     // TODO - remove the `|| ...` when done with development
     return this.model.get('account') || this.user.getSignedInAccount();
   }
 
+  gotoProductPage() {
+    this.window.location.href = SmsSendView.FXA_PRODUCT_PAGE_URL;
```

Comment:
Can you just set the href of the anchor element? The [ExternalLinksMixin](https://github.com/mozilla/fxa/blob/3bbffd84da7139af2b6d1593141a1171f7c528ab/packages/fxa-content-server/app/scripts/views/mixins/external-links-mixin.js) is added to BaseView, it'll take care of flushing metrics for you and you won't need this extra code. Direct assignment of `window.location.href` like this doesn't take care of flushing metrics.